criu/files.c

```diff
@@ -1332,6 +1332,8 @@ int prepare_fds(struct pstree_item *me)
 
 static int fchroot(int fd)
 {
+	if (!has_cap_sys_chroot(opts.cap_eff))
```

Comment:
We need to add a comment that explain this. Right now, it looks very suspicious...

Reply:
Not sure about it myself. @adrianreber it looks to me like in unprivileged mode we skip restoring the chroot if we don't have the cap for it without signaling any errors. Is this because we are assuming that the process cannot be chrooted under these conditions and this step will be a no-op? Should we be paranoid and check that we're not restoring a process outside of a chroot jail?